### ISSUE TITLE
Build failure with default buildtools.json in VS for Mac

### ISSUE THREAD

**Issue by bijington** [dansiegel/Mobile.BuildTools#263]
# Description

Installing package `2.0.226-pre-g0393c9afa6` fails a build with:

```exception
/Mobile.BuildTools.targets(5,5): Error: Method not found: bool Newtonsoft.Json.Serialization.JsonProperty.get_IsRequiredSpecified() (SuperWordsearch)
```

Note that this works in version `2.0.208-pre-ge7c596f789` and also with the version above using the CLI.

## Reproduction Steps

1. Install or update to `2.0.226-pre-g0393c9afa6`
2. Build
3. Observe the error

## Expected Behavior

The projects should compile.

## Actual Behavior

```exception
/Mobile.BuildTools.targets(5,5): Error: Method not found: bool Newtonsoft.Json.Serialization.JsonProperty.get_IsRequiredSpecified() (SuperWordsearch)
```

## Environment

- OS: macOS (11.4)
- IDE: VSMac
- BuildTools Version: `2.0.226-pre-g0393c9afa6`
- IDE Version: 8.10.4 (build 11)
- Build Target Framework: netstandard2.0 | Xamarin.iOS | Xamarin.Android

## Reproduction App

Sample project that fails inside VS for Mac.
[template.zip](https://github.com/dansiegel/Mobile.BuildTools/files/6766146/template.zip)

Frustratingly it seems to build find in the CLI.
[msbuild.binlog.zip](https://github.com/dansiegel/Mobile.BuildTools/files/6766143/msbuild.binlog.zip)

**Comment by bijington** [comment]
Could it relate to this addition:

`<PackageReference Update="Newtonsoft.Json.Schema" Version="3.0.14" />`

A potential conflict with whatever VS for Mac is loading?

**Comment by dansiegel** [comment]
closed by #264
